src/pages/account/mijn-materiaalaanvragen/index.tsx

```diff
@@ -22,13 +22,24 @@ import { renderOgTags } from '@shared/helpers/render-og-tags';
 import useTranslation from '@shared/hooks/use-translation/use-translation';
 import { DefaultSeoInfo } from '@shared/types/seo';
 
-import { useGetMaterialRequests } from '@material-requests/hooks/get-material-requests';
+import MaterialRequestDetailBlade from '../../../modules/account/components/MaterialRequestDetailBlade/MaterialRequestDetailBlade';
+
+import {
+	useGetMaterialRequestById,
+	useGetMaterialRequests,
+} from '@material-requests/hooks/get-material-requests';
 import { MaterialRequest, MaterialRequestKeys } from '@material-requests/types';
 import { VisitorLayout } from 'modules/visitors';
 
 const AccountMyMaterialRequests: NextPage<DefaultSeoInfo> = ({ url }) => {
 	const { tHtml, tText } = useTranslation();
 	const [filters, setFilters] = useQueryParams(ACCOUNT_MATERIAL_REQUESTS_QUERY_PARAM_CONFIG);
+	const [isDetailBladeOpen, setIsDetailBladeOpen] = useState(false);
+	const [currentMaterialRequest, setCurrentMaterialRequest] = useState<MaterialRequest>();
+
+	const { data: currentMaterialRequestDetail, isFetching: isLoading } = useGetMaterialRequestById(
+		currentMaterialRequest?.id || ''
```

Comment:
better to explicitly pass null instead of empty string and make the useGetMaterialRequestById function accept string | null